andela/caret-bn-frontend
# #169889384-Handle User Verification

#### What does this PR do?
* Handle User Verification
#### Description of Task to be completed?
* Created component userVerify
* Created test for userVerify component
#### How should this be manually tested?
* Clone ft-user-verification-169889384
* run npm i
* run npm run dev
* navigate to postman
* Register a new user
* after making registration go in your email
* click on verify button

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#169889384
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38130305/70146715-71a3cb80-16ab-11ea-9516-7e34cf5af3ee.png)

![image](https://user-images.githubusercontent.com/38130305/70444259-49004500-1aa2-11ea-9488-01858f40fe44.png)

![image](https://user-images.githubusercontent.com/38130305/70444239-4140a080-1aa2-11ea-967c-29dcfd1d68bf.png)

#### Questions:
N/A
